graphene_django/fields.py

```diff
@@ -154,18 +160,61 @@ def resolve_connection(cls, connection, args, iterable, max_limit=None):
         ):
             args["first"] = max_limit
 
+        if graphene_settings.USE_DATALOADERS:
+            if not hasattr(info.context, "dataloaders"):
+                info.context.dataloaders = {}
```

Comment:
Would using `setattr` be a more canonical way of adding a property to an existing object?